custom/conf/app.ini.sample

```diff
@@ -36,6 +36,8 @@ DISABLE_HTTP_GIT = false
 ACCESS_CONTROL_ALLOW_ORIGIN = 
 ; Force ssh:// clone url instead of scp-style uri when default SSH port is used
 USE_COMPAT_SSH_URI = false
+; Close issues as long as a commit on any branch marks it as fixed
+CLOSE_ISSUES_VIA_COMMITS_IN_ANY_BRANCH = false
```

Comment:
imho this should not be server global value

Reply:
Oops, I thought a global config was what was needed.

So 2 ways:

- So this should be set per repository but I should provide a 
migration that will update the field based on this new config value? 

- Or I should just get rid of this config value, provide a migration with 
a new field and set it all to false?

I am fine with either of them or leaving it as a global config. I just need a 
 ___go ahead___ here.

Reply:
I'm fine with both options as long as it could be set per-repository. First option probably covers more scenarios as default can be set in config

Reply:
Done @lafriks ..

Reply:
Done @lafriks.